core/constants.yaml

```diff
@@ -121,31 +123,38 @@ qanet: &qanet
 
   recoveryHeadersMessage: 100
 
+  genesisBinSuffix: qa
+
 testnet: &testnet
   <<: *prod_base
 
-  k: 60
+  k: 72
```

Comment:
This change is not related to this PR, it just adjusts testnet `k` to be such that epoch is one day (this is used in modern testnet staging)